Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Subject: fix hyphen vs. minus in man pages
Origin: http://svn.asterisk.org/svn/asterisk/trunk@256704 
    
In troff '-' is used for a hyphen. A minus is denoted by '\-' . This is
normally also used for a dash.

This patch converts all '-'-s that are minuses or dashes to '\-'.

diff --git a/contrib/scripts/astgenkey.8 b/contrib/scripts/astgenkey.8
index 328a4d2..063feca 100644
--- a/contrib/scripts/astgenkey.8
+++ b/contrib/scripts/astgenkey.8
@@ -53,12 +53,11 @@
 .ta 8n 16n 24n 32n 40n 48n 56n 64n 72n  
 .TH ASTGENKEY 8 "May 14th, 2005" "Asterisk" "Linux Programmer's Manual"
 .SH NAME
-.B astgenkey
--- generates keys for for Asterisk IAX2 RSA authentication
+.B astgenkey \- generates keys for for Asterisk IAX2 RSA authentication
 .SH SYNOPSIS
 .PP 
 .B astgenkey
-[ -q ] [ -n ] [ \fIkeyname\fP ]
+[ \-q ] [ \-n ] [ \fIkeyname\fP ]
 
 .SH DESCRIPTION
 .B astgenkey 
@@ -66,7 +65,7 @@ This script generates an RSA private and public key pair in PEM format
 for use by Asterisk.  The private key should be kept a secret, as it can 
 be used to fake your system's identity.  Thus by default (without the 
 option 
-.I -n
+.I \-n
 ) the script will create a passphrase-encrypted copy of your secret key: 
 without entering the passphrase you won't be able to use it. 
 
@@ -99,12 +98,12 @@ show keys
 .RE
 
 .SH OPTIONS
-.B -q
+.B \-q
 .RS
 Run quietly.
 .RE
 
-.B -n
+.B \-n
 .RS
 Don't encrypt the private key.
 .RE
@@ -132,7 +131,7 @@ user "asterisk").
 .SH "SEE ALSO" 
 asterisk(8), genrsa(1), rsa(1), 
 
-http://www.voip-info.org/wiki-Asterisk+iax+rsa+auth
+http://www.voip\-info.org/wiki\-Asterisk+iax+rsa+auth
 
 .SH "AUTHOR" 
 This manual page was written by Tzafrir Cohen <tzafrir.cohen@xorcom.com> 
@@ -141,4 +140,4 @@ the terms of the GNU General Public License, Version 2 any
 later version published by the Free Software Foundation. 
 
 On Debian systems, the complete text of the GNU General Public 
-License can be found in /usr/share/common-licenses/GPL. 
+License can be found in /usr/share/common\-licenses/GPL\-2. 
diff --git a/contrib/scripts/autosupport.8 b/contrib/scripts/autosupport.8
index 50949e9..ed4d4a8 100644
--- a/contrib/scripts/autosupport.8
+++ b/contrib/scripts/autosupport.8
@@ -37,4 +37,4 @@ the terms of the GNU General Public License, Version 2 any
 later version published by the Free Software Foundation. 
 
 On Debian systems, the complete text of the GNU General Public 
-License can be found in /usr/share/common-licenses/GPL. 
+License can be found in /usr/share/common\-licenses/GPL\-2. 
diff --git a/contrib/scripts/safe_asterisk.8 b/contrib/scripts/safe_asterisk.8
index ebd9514..b5395cd 100644
--- a/contrib/scripts/safe_asterisk.8
+++ b/contrib/scripts/safe_asterisk.8
@@ -14,7 +14,7 @@ fear asterisk may crash.
 
 The script does not run in the background like a standard service. Rather, 
 it runs in its own linux virtual console (9, by default).
-It also uses the option '-c' of asterisk(8) to avoid detaching asterisk 
+It also uses the option '\-c' of asterisk(8) to avoid detaching asterisk 
 from that terminal.
 
 safe_asterisk also runs asterisk with unlimited core file size, and thus 
@@ -66,4 +66,4 @@ the terms of the GNU General Public License, Version 2 any
 later version published by the Free Software Foundation. 
 
 On Debian systems, the complete text of the GNU General Public 
-License can be found in /usr/share/common-licenses/GPL. 
+License can be found in /usr/share/common\-licenses/GPL\-2. 
diff --git a/doc/asterisk.8 b/doc/asterisk.8
index f3eb1e9..b4f7026 100644
--- a/doc/asterisk.8
+++ b/doc/asterisk.8
@@ -9,10 +9,10 @@
 asterisk \- All-purpose telephony server.
 .SH SYNOPSIS
 
-\fBasterisk\fR [ \fB-tThfdvVqpRgciIn\fR ] [ \fB-C \fIfile\fB\fR ] [ \fB-U \fIuser\fB\fR ] [ \fB-G \fIgroup\fB\fR ] [ \fB-x \fIcommand\fB\fR ] [ \fB-M \fIvalue\fB\fR ]
+\fBasterisk\fR [ \fB\-tThfdvVqpRgciIn\fR ] [ \fB\-C \fIfile\fB\fR ] [ \fB\-U \fIuser\fB\fR ] [ \fB\-G \fIgroup\fB\fR ] [ \fB\-x \fIcommand\fB\fR ] [ \fB\-M \fIvalue\fB\fR ]
 
 
-\fBasterisk -r\fR [ \fB-v\fR ] [ \fB-x \fIcommand\fB\fR ] [ \fB-s \fIsocket\fB\fR ]
+\fBasterisk \-r\fR [ \fB\-v\fR ] [ \fB\-x \fIcommand\fB\fR ] [ \fB\-s \fIsocket\fB\fR ]
 
 .SH "DESCRIPTION"
 .PP
@@ -32,7 +32,7 @@ Virtually all aspects of the operation of asterisk's configuration files
 can be found in the sample configuration files.  The format for those files
 is generally beyond the scope of this man page.
 .PP
-When running with \fB-c\fR, \fB-r\fR or \fB-R\fR
+When running with \fB\-c\fR, \fB\-r\fR or \fB\-R\fR
 options, Asterisk supplies a powerful command line, including command
 completion, which may be used to monitors its status, perform a variety
 of administrative actions and even explore the applications that are
@@ -41,161 +41,161 @@ currently loaded into the system.
 Asterisk is a trademark of Digium, Inc.
 .SH "OPTIONS"
 .TP
-\fB-C \fIfile\fB\fR
+\fB\-C \fIfile\fB\fR
 Use \fIfile\fR as master configuration file
 instead of the default, /etc/asterisk/asterisk.conf
 .TP
-\fB-c\fR
+\fB\-c\fR
 Provide a control console on the calling terminal.
 Specifying this option implies \fB-f\fR and will cause
 asterisk to no longer fork or detach from the controlling terminal.
 .TP
-\fB-d\fR
+\fB\-d\fR
 Enable extra debugging statements.
 
 Note: This always sets the debug level in the asterisk process,
 even if it is running in the background. This will affect the size
 of your log files.
 .TP
-\fB-f\fR
+\fB\-f\fR
 Do not fork or detach from controlling terminal.
 .TP
-\fB-g\fR
+\fB\-g\fR
 Remove resource limit on core size, thus forcing Asterisk to dump
 core in the unlikely event of a segmentation fault or abort signal.
 \fBNOTE:\fR in some cases this may be incompatible
-with the \fB-U\fR or \fB-G\fR flags.
+with the \fB\-U\fR or \fB\-G\fR flags.
 .TP
-\fB-G \fIgroup\fB\fR
+\fB\-G \fIgroup\fB\fR
 Run as group \fIgroup\fR instead of the
 calling group.  \fBNOTE:\fR this requires substantial work
 to be sure that Asterisk's environment has permission to write
 the files required for its operation, including logs, its comm
 socket, the asterisk database, etc.
 .TP
-\fB-h\fR
+\fB\-h\fR
 Provide brief summary of command line arguments and terminate.
 .TP
-\fB-i\fR
+\fB\-i\fR
 Prompt user to initialize any encrypted private keys for IAX2
 secure authentication during startup.
 .TP
-\fB-L \fIloadaverage\fB\fR
+\fB\-L \fIloadaverage\fB\fR
 Limits the maximum load average before rejecting new calls.  This can
 be useful to prevent a system from being brought down by terminating
 too many simultaneous calls.
 .TP
-\fB-m\fR
-Disable log and verbose output to remote (-r) consoles.
+\fB\-m\fR
+Disable log and verbose output to remote (\-r) consoles.
 .TP
 \fB-M \fIvalue\fB\fR
 Limits the maximum number of calls to the specified value.  This can
 be useful to prevent a system from being brought down by terminating
 too many simultaneous calls.
 .TP
-\fB-n\fR
+\fB\-n\fR
 Disable ANSI colors even on terminals capable of displaying them.
 .TP
-\fB-p\fR
+\fB\-p\fR
 If supported by the operating system (and executing as root),
 attempt to run with realtime priority for increased performance and
 responsiveness within the Asterisk process, at the expense of other
 programs running on the same machine.
 .TP
-\fB-q\fR
+\fB\-q\fR
 Reduce default console output when running in conjunction with
-console mode (\fB-c\fR).
+console mode (\fB\-c\fR).
 .TP
-\fB-r\fR
+\fB\-r\fR
 Instead of running a new Asterisk process, attempt to connect
 to a running Asterisk process and provide a console interface
 for controlling it.
 .TP
-\fB-R\fR
-Much like \fB-r\fR\&.  Instead of running a new Asterisk process, attempt to connect
+\fB\-R\fR
+Much like \fB\-r\fR\&.  Instead of running a new Asterisk process, attempt to connect
 to a running Asterisk process and provide a console interface
 for controlling it. Additionally, if connection to the Asterisk 
 process is lost, attempt to reconnect for as long as 30 seconds.
 .TP
-\fB-s \fIsocket\fB\fR
+\fB\-s \fIsocket\fB\fR
 Allows to specify the socket file to be used to connect to the
 Asterisk console. Used in conjunction with \fB-r\fR or \fB-R\fR.
 .TP
-\fB-I\fR
+\fB\-I\fR
 Enable internal timing if DAHDI timer is available
 The default behaviour is that outbound packets are phase locked
 to inbound packets. Enabling this switch causes them to be
 locked to the internal DAHDI timer instead.
 .TP
-\fB-t\fR
+\fB\-t\fR
 When recording files, write them first into a temporary holding directory, 
 then move them into the final location when done.
 .TP
-\fB-T\fR
+\fB\-T\fR
 Add timestamp to all non-command related output going to the console
 when running with verbose and/or logging to the console.
 .TP
-\fB-U \fIuser\fB\fR
+\fB\-U \fIuser\fB\fR
 Run as user \fIuser\fR instead of the
 calling user.  \fBNOTE:\fR this requires substantial work
 to be sure that Asterisk's environment has permission to write
 the files required for its operation, including logs, its comm
 socket, the asterisk database, etc.
 .TP
-\fB-v\fR
+\fB\-v\fR
 Increase the level of verboseness on the console.  The more times
 \fB-v\fR is specified, the more verbose the output is.
-Specifying this option implies \fB-f\fR and will cause
+Specifying this option implies \fB\-f\fR and will cause
 asterisk to no longer fork or detach from the controlling terminal.
-This option may also be used in conjunction with \fB-r\fR
-and \fB-R\fR\&.
+This option may also be used in conjunction with \fB\-r\fR
+and \fB\-R\fR\&.
 
 Note: This always sets the verbose level in the asterisk process,
 even if it is running in the background. This will affect the size
 of your log files.
 .TP
-\fB-V\fR
+\fB\-V\fR
 Display version information and exit immediately.
 .TP
-\fB-W\fR
+\fB\-W\fR
 Change the terminal colors to compensate for a light background,
 rather than a dark background, as is the default.
 .TP
-\fB-x \fIcommand\fB\fR
+\fB\-x \fIcommand\fB\fR
 Connect to a running Asterisk process and execute a command on
 a command line, passing any output through to standard out and
 then terminating when the command execution completes.  Implies
-\fB-r\fR when \fB-R\fR is not explicitly
+\fB\-r\fR when \fB\-R\fR is not explicitly
 supplied.
 .SH "EXAMPLES"
 .PP
-\fBasterisk\fR - Begin Asterisk as a daemon
+\fBasterisk\fR \- Begin Asterisk as a daemon
 .PP
-\fBasterisk -vvvgc\fR - Run on controlling terminal
+\fBasterisk \-vvvgc\fR \- Run on controlling terminal
 .PP
-\fBasterisk -rx "core show channels"\fR - Display channels on running server
+\fBasterisk \-rx "core show channels"\fR \- Display channels on running server
 .SH "BUGS"
 .PP
 Bug reports and feature requests may be filed at http://bugs.digium.com
 .SH "SEE ALSO"
 .PP
-*CLI> \fBhelp\fR - Help on Asterisk CLI
+*CLI> \fBhelp\fR \- Help on Asterisk CLI
 .PP
-*CLI> \fBcore show applications\fR - Show loaded dialplan applications
+*CLI> \fBcore show applications\fR \- Show loaded dialplan applications
 .PP
-*CLI> \fBcore show functions\fR - Show loaded dialplan functions
+*CLI> \fBcore show functions\fR \- Show loaded dialplan functions
 .PP
-*CLI> \fBdialplan show\fR - Show current dialplan
+*CLI> \fBdialplan show\fR \- Show current dialplan
 .PP
-http://www.asterisk.org - The Asterisk Home Page
+http://www.asterisk.org \- The Asterisk Home Page
 .PP
-http://www.asteriskdocs.org - The Asterisk Documentation Project
+http://www.asteriskdocs.org \- The Asterisk Documentation Project
 .PP
-http://www.voip-info.org/wiki-Asterisk - The Asterisk Wiki
+http://www.voip\-info.org/wiki-Asterisk \- The Asterisk Wiki
 .PP
-http://www.digium.com/ - Asterisk sponsor and hardware supplier
+http://www.digium.com/ \- Asterisk sponsor and hardware supplier
 .PP
-http://www.markocam.com/ - Asterisk author's web cam
+http://www.markocam.com/ \- Asterisk author's web cam
 .SH "AUTHOR"
 .PP
 Mark Spencer <markster@digium.com>
